docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+services:
+  mysql:
+    image: mysql:8.0
+    restart: always
+    ports:
+      - 3306:3306
+      - 33060:33060
+    environment:
+      MYSQL_DATABASE: "messanger_project"
+      MYSQL_PASSWORD: ensasha2002
+      MYSQL_ROOT_PASSWORD: ensasha2002
+    command: mysqld --disable-ssl --sql_mode=""
```

Comment:
The `docker-compose.yml` file is not required, it's needed for running mysql in a docker container using a single simple command:
`docker-compose up`